<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>923</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>923</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Html5 Checker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/recursos/imagenes/logofinal.png</normaloff>:/recursos/imagenes/logofinal.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
border-image: url(:/recursos/imagenes/fondonexus.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="AppOutput">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>78</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));


border-image: url(:/imagenes/negro4.gif);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbUbicacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>525</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));


border-image: url(:/imagenes/negro4.gif);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ubicaci√≥n:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>491</width>
      <height>403</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));


border-image: url(:/imagenes/negro4.gif);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>272</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));


border-image: url(:/imagenes/negro4.gif);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recursos/imagenes/banner.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>6</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/recursos/imagenes/file_add.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAbrir_2">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>6</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/recursos/imagenes/close.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/recursos/imagenes/logofinal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
